tests/engines/registries/acceptance/draft/draft-test.ts

```diff
@@ -141,6 +143,7 @@ module('Registries | Acceptance | draft form', hooks => {
 
         // Navigate to review
         await click('[data-test-link="review"]');
+        await percySnapshot(assert);
```

Comment:
Percy uses the assert object (which has the test name) as name for the snapshot. When you have multiple `percySnapshot`s in one test, Percy prefers unique names for each snapshot. See example [here](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/tests/acceptance/dashboard-test.ts#L415)